### ISSUE TITLE
Migration of Class Libraries not available for Net48 projects using SDK project system

### ISSUE THREAD

**Issue by jjmanton** [dotnet/systemweb-adapters#255]
### Describe the bug
When I attempt to migrate class libraries from Visual Studio that use the SDK style project, I found that the menu option is not available, even though the `TargetFramework` of these projects is `net48`. 

### To Reproduce
1. Create a new net48 console project in Visual Studio
2. Create a class library, targeting net48 as well
3. Note that migration option is available in Visual Studio
4. Convert the class library to SDK project style, but keep the net48 TFM
5. Note Migration is no longer available

### Exceptions (if any)
n/a

### Further technical details 

Please include the following if applicable:

*ASP.NET Framework Application*:
- Technologies and versions used (i.e. MVC/WebForms/etc): N/A
- .NET Framework Version: 4.8
- IIS Version: Latest
- Windows Version: Latest

*ASP.NET Core Application*:
- Targeted .NET version: 6
- .NET SDK version: 6

**Comment by adityamandaleeka** [comment]
@abpiskunov

**Comment by abpiskunov** [comment]
Yes, currently we only migrate "classic" class libraries toward SDK style. SDK project upgrades would come in next preview.

**Comment by jjmanton** [comment]
In the mean time, is there any way to run the transformers individually, possibly from a command line? If I recall running the migration on a class, the output shows a few different (maybe 6-7) operations that are performed on the class.

**Comment by abpiskunov** [comment]
There no command line interface that can run this new experience and it's contracts at the moment. We would enable cli some time later when VS experience is ready. However, you have a good point of running only selected transformers, which is agnostic to CLI or VS , we should support that at some point.
